java/AxivityReader.java

```diff
@@ -306,22 +322,24 @@ private static void readCwaBuffer(ByteBuffer buf, LocalDateTime SESSION_START,
                     x = xRaw / 256.0;
                     y = yRaw / 256.0;
                     z = zRaw / 256.0;
-                    timeValue = getEpochMillis(blockTime.plusNanos(START_OFFSET_NANOS));
+                    t = zonedWithDSTCorrection(blockTime).toInstant().toEpochMilli();
```

Comment:
This is where the correction is called. blockTime which is a LocalDateTime object (no timezone) is first zoned with DST correction (see zonedWithDSTCorrection()), then converted to Unix time in milliseconds (milliseconds since 1 Jan 1970 UTC). NOW it is truly a Unix time!

Reply:
@angerhang - we may have to do this for the ActigraphReader.java class too?